forch/port_state_manager.py

```diff
@@ -207,7 +216,8 @@ def _handle_authenticated_device(self, mac, device_behavior, static):
             device_behaviors.setdefault(mac, DeviceBehavior()).CopyFrom(device_behavior)
 
             static_port_behavior = self._static_port_behaviors.get(mac)
-            if not self._sequester_segment or static_port_behavior == PortBehavior.cleared:
+            sequestering_allowed = _clean_mac(mac) in self._sequester_allowed_list
```

Comment:
One bit of functionality we need to control is the default behavior.  E.g., depending on configuration, a device that is listed in the set will either automatically be sequestered/tested or not.  The nominal case is that everything would be sequestered, this provision is to incrementally enable sequestering for controlled situations -- but we shouldn't get into a state where everything has to be listed always to be tested.

The easy thing to do here (IMHO) is in combination with the map concept below, just look up the "enabled" property in the map if the device is listed, and if it's not then go with a (configurable) default... or something like that.

Reply:
I had a bool flag for indicating whether to only test the list of devices but that's not pushed. Isn't that cleaner than a map? When other options are needed, those can go in a separate map under sequester config /  somewhere else that's uncoupled from the target macs.

Reply:
Is `allowed_list` functionally the same with `PortBehavior`?

We can explicitly set a device into the desired state (sequester, operational, etc.) by configuring the device's port behavior in the static behavior file.

https://github.com/faucetsdn/forch/blob/ae89a7776f687737f8e7c92e5dce8972768e18bb/proto/shared_constants.proto#L165

Reply:
This PR is obsolete now. There’s another one that’s merged.